### PR TITLE
Release/781xx 

### PR DESCRIPTION
**Pull request by rkapoor10**
## SUMMARY

Release 781xx of Harness CD Community Edition

### Updates 
- docker-compose.yml
- values.yaml
- README.md

### Bug Fixes
Not able to sign-up user with new installation

### CODE REVIEW

File: docker-compose/harness/README.md [hunk 1]
```diff
@@ -22,11 +22,20 @@ This page contains the instructions to run Harness CD Community Edition using Do
    ```shell
    docker-compose up -d
    ```
+   ![compose-up](../../static/compose-up.png)
```
Comment (Ompragash) [inline docker-compose/harness/README.md:25]: do we really need screenshots for the basic Docker compose commands here?

File: docker-compose/harness/README.md [hunk 2]
```diff
@@ -47,6 +56,19 @@ For example,
...
+You may also consider the following - 
+
+```shell 
+docker ps
+```
+
+#### View logs of processes
+```shell
+docker logs -f <CONTAINER ID>
+```
```
Comment (Ompragash) [inline docker-compose/harness/README.md:68]: Running the `docker-compose logs` command in the directory with docker-compose.yml shows all the current logs so we don't need `docker ps & logs -f <CONTAINER ID>`

**Review by Ompragash — CHANGES_REQUESTED** [review]

**Comment by Ompragash** [comment]
For the Helm installer, the `README.md` is under **helm** directory, and then a sub-directory for **harness** with `values.yaml` file inside it. 

But, for the Docker installer, the `README.md` is under `docker-compose/harness` directory.

Maybe it was mistakenly added in there initially and needs a little re-arrangement.

**Comment by rkapoor10** [comment]
> For the Helm installer, the `README.md` is under **helm** directory, and then a sub-directory for **harness** with `values.yaml` file inside it.
> 
> But, for the Docker installer, the `README.md` is under `docker-compose/harness` directory.
> 
> Maybe it was mistakenly added in there initially and needs a little re-arrangement.

We can move out README.md file one level up which is under `docker-compose/harness` directory currently to maintain consistency.

**Comment by Ompragash** [comment]
> We can move out README.md file one level up which is under `docker-compose/harness` directory currently to maintain consistency.

Also, test the compose commands and make sure everything works and if required make the appropriate changes

**Review by harnesspravin — APPROVED** [review]
/lgtm

**Review by Ompragash — APPROVED** [review]

**Pull request merged**